src/components/CampaignForm.tsx

```diff
@@ -163,7 +163,7 @@ const CampaignForm = ({ onSubmit, onCancel, campaign }: CampaignFormProps): JSX.
     const handleSubmit = (values: Record<string, string>) => {
         const { info_text, status, title, visibility, manager_id } = values;
         onSubmit({
-            ...info_text ? { info_text } : null,
+            info_text: info_text == undefined ? '' : info_text,
```

Comment:
I would just have done `info_text: info_text || ''`. Would that work?

Reply:
that's fair, yes!